suite/tests/CMakeLists.txt

```diff
@@ -3751,7 +3751,7 @@ endforeach(run)
 # A subset of release tests has been added and labeled with RUN_IN_RELEASE. It focuses on
 # Google's use of the drcachesim client. We do not want to extend the Travis regression
 # time by too much and keep the following list concise.
-if (NOT DEBUG AND UNIX AND X64 AND NOT CMAKE_COMPILER_IS_CLANG)
+if (NOT DEBUG AND UNIX AND X64 AND X86 AND NOT CMAKE_COMPILER_IS_CLANG)
```

Comment:
> Excludes AArch64 from running release-external-64 tests on Travis.

This does not sound like the right solution.  A RUN_IN_RELEASE tag should be cross-platform. 
 Just use the name `common.${control_flags}`.

Reply:
Ok, that's better. Thanks.